js/dialogcards.js

```diff
@@ -853,6 +1082,57 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
     }
   };
 
+  /**
+   * Remove card from DOM and from cards stack after user has checked the "gotit" button.
+   */
+
+  C.prototype.gotIt = function ($card) {
```

Comment:
The function seems to:
1. trigger xAPI
2. navigate to a new card if possible
3. if not possible a procedure for finishing up starts
4. Some maintainance

Maybe some of this (2 and 3 perhaps) could be separate functions?